core.serge.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "core",
+  "parser_plugin": {
+    "plugin": "parse_js"
+  },
+  "source_match": "en\\.js$",
+  "source_dir": "lang",
+  "output_file_path": "lang/%LANG%.js",
+  "output_lang_rewrite": [
+    "ar-sa ar",
```

Comment:
I know this is not special for this PR, but how does this array work?  i.e. I see a culture-specific locale and a non-culture-specific locale separated by a space - how do they get used?

Reply:
So I think it's saying take these translations for this more specific locale code (e.g., ar-sa) and put them in a file called "ar" instead because that's what we're expecting. I do need to fix some of these based on the files we have here.